### PR TITLE
Dissolve shader effect

### PR DESCRIPTION
**Pull request by recklol**
Forum thread:
https://forum.unity.com/threads/burn-disappear-effect.997856/
Card:
https://open.codecks.io/unity-open-project-1/decks/24/card/176-burn-disappear-effect

This is a basic, stylized implementation of a dissolve effect. A shader and a Particle System working together. 
These changes are important to give feedback to the player, and also feel satisfying to look at.
I implemented it the best way I could, making it customizable easily.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+UOP1_Project/Packages/packages-lock.json
+UOP1_Project/ProjectSettings/PackageManagerSettings.asset
+UOP1_Project/ProjectSettings/Packages/com.unity.polybrush/Settings.json
+UOP1_Project/ProjectSettings/Packages/com.unity.probuilder/Settings.json
+UOP1_Project/ProjectSettings/Packages/com.unity.progrids/Settings.json
+UOP1_Project/ProjectSettings/ProjectVersion.txt
+UOP1_Project/ProjectSettings/URPProjectSettings.asset
```
Comment (ciro-unity) [inline .gitignore:8]: Why would you want to ignore all these?

File: UOP1_Project/Assets/Scenes/CharController.unity [hunk 2]
```diff
@@ -188,7 +188,7 @@ Transform:
   m_Children:
```
Comment (ciro-unity) [inline UOP1_Project/Assets/Scenes/CharController.unity:188]: Why all the changes in this file?

**Review by ciro-unity — CHANGES_REQUESTED** [review]
Great PR, but there are a few files which I don't think belong in it. I understand the need to demo the effect in the art scene, but you can keep it in that one so it doesn't conflict with changes on other scenes.

Can you strip everything that is not absolutely necessary? Thanks!

**Comment by ciro-unity** [comment]
Hope everything is good with this one! Come ask for help on the forums if you're having trouble with Git?

**Comment by recklol** [comment]
I messed up my local repo AND also had other problems with my computer. I made another branch, where I have a clean commit with only the necessary files.

Should I make a new pullrequest from that?

**Comment by treivize** [comment]
Hello @recklol,

Just to let you know that to progress on delivering your dissolve effect I have opened a new PR to bring it back on an the current code stack. Please do not hesitate to review my changes and let me know if you are fine or now with them.
[new PR](https://github.com/UnityTechnologies/open-project-1/pull/217)

**Pull request closed without merging**